docs/panels.rst

```diff
@@ -9,6 +9,14 @@ Default built-in panels
 
 The following panels are enabled by default.
 
+History
+~~~~~~~
+
+Path: ``debug_toolbar.panels.history.VersionsPanel``
```

Comment:
Typo? (`VersionsPanel`)

Reply:
Yeah, copy paste error. Thanks for catching it.